src/Fieldtypes/Entries.php

```diff
@@ -149,8 +150,11 @@ protected function getCreatables()
 
         $collections = $this->getConfiguredCollections();
 
-        return collect($collections)->flatMap(function ($collection) use ($collections) {
-            $collection = Collection::findByHandle($collection);
+        return collect($collections)->flatMap(function ($collectionHandle) use ($collections) {
+            $collection = Collection::findByHandle($collectionHandle);
+
+            throw_if(!$collection, new CollectionNotFoundException($collectionHandle));
```

Comment:
Really nitpicking now: this should be `! $collection` with a space, to make StyleCI stop complaining and everything pass.

Reply:
Done! Gotta keep StyleCI happy.